addons/mail/static/src/widgets/form_renderer/form_renderer_tests.js

```diff
@@ -426,6 +426,100 @@ QUnit.test('chatter should become enabled when creation done', async function (a
     );
 });
 
+QUnit.test('read more/less links are correctly added and toggled', async function (assert) {
```

Comment:
Name of test could still be improved (_correctly_ bears a lot of context that is not explicit neither in name of test nor in comment as header of test content).

Current test verifies read more is consistent when changing from readonly to edit mode, and also verifies that it becomes read less after click. By stating it like this, it feels more natural to split this into 2 tests.